experimental/PropertyDDS/packages/property-inspector-table/src/utils.tsx

```diff
@@ -630,8 +630,8 @@ export const collectionChildTableRow = (collectionPropertyProxy: BaseProxifiedPr
         property = (referencedPropertyParent as ContainerProperty).get(relativePathFromParent)!;
         if (property) {
           if (PropertyFactory.instanceOf(property as BaseProperty, "BaseProperty")) {
```

Comment:
Could this be updated to be a type guard, so if it returns true then TS can know that `property` is `BaseProperty`?  (not in this PR)